docker/cassandra/default-config/cassandra.yaml

```diff
@@ -1,13 +1,6 @@
-# Cassandra storage config YAML
+data_file_directories:
```

Comment:
Not sure what we should consider as the "defaults" here.

Setting the directories -- yes, but other things? I guess whatever the out-of-box settings are...

Reply:
The way C* handles defaults is complex: with the some config options having defaults in the `Config` class, others requiring something to be set in the YAML for C* to start, and others where the code default and the YAML default differ...

I might leave the package distro `cassandra.yaml` in `/etc/cassandra` and everything else can be done as overides.